docs/application/dotnet/guides/account/data-sync.md

```diff
@@ -54,21 +54,21 @@ To enable your application to use the synchronization management functionality:
 
 2.  To use the methods and properties of the Tizen.Account.SyncManager namespace, include it in your application:
 
-    ```
+    ```csharp
     using Tizen.Account.SyncManager;
     ```
 
 > **Note**   
-> To use the features of the [Tizen.Account.SyncManager](https://samsung.github.io/TizenFX/latest/api/Tizen.Account.SyncManager.html) namespace, the service application must first [set the callbacks](#set_callback). A UI application cannot initialize or set callback methods through the [Tizen.Account.SyncManager.SyncAdapter](https://samsung.github.io/TizenFX/latest/api/Tizen.Account.SyncManager.SyncAdapter.html) class. Instead, the UI application must call the methods of the [Tizen.Account.SyncManager.SyncClient](https://samsung.github.io/TizenFX/latest/api/Tizen.Account.SyncManager.SyncClient.html) class to request sync operations from the service application.
+> To use the features of the [Tizen.Account.SyncManager](/application/dotnet/api/TizenFX/latest/api/Tizen.Account.SyncManager.html) namespace, the service application must first [set the callbacks](#set_callback). A UI application cannot initialize or set callback methods through the [Tizen.Account.SyncManager.SyncAdapter](/application/dotnet/api/TizenFX/latest/api/Tizen.Account.SyncManager.SyncAdapter.html) class. Instead, the UI application must call the methods of the [Tizen.Account.SyncManager.SyncClient](/application/dotnet/api/TizenFX/latest/api/Tizen.Account.SyncManager.SyncClient.html) class to request sync operations from the service application.
 
 <a name="set_callback"></a>
 ## Setting Sync Adapter Callbacks
```

Comment:
Global: Sentence case for headings
```suggestion
## Set sync adapter callbacks

Reply:
See https://github.com/Samsung/tizen-docs/pull/1387#discussion_r575045312